models/incexp.json

```diff
@@ -1,7 +1,8 @@
 {
     "fact_table": "incexp_facts",
     "label": "Income and Expenditure",
-    "description": "Statement of financial performance. This is based on the OSB, OSR, OSA and OSAA return forms.",
+    "description": "Statement of Financial Position. How a municipality has spent money and received income.",
```

Comment:
_Financial Position_ is Balance Sheet.
_Financial Performance_ is Operating Income and Expenditure